rake_config/vanishing_data.rb

```diff
@@ -0,0 +1,74 @@
+class SourceCSV
+  def initialize(str)
+    @raw = str
+  end
+
+  def ids
+    grouped.keys
+  end
+
+  def rows(*kk)
+    kk.map { |k| grouped[k].first }
```

Comment:
Good catch. I think there are likely to be all sorts of odd edge cases when we're dealing with multiple memberships — e.g. only some of them disappearing. But I think we should defer handling those until we actually see them. This one is an easy fix though, so I've added it.